models/reporting/olids/data_quality/dq_eosinophil_count_issues.sql

```
@@ -31,17 +31,17 @@ SELECT
 
     CASE
         WHEN is_negative THEN 'Negative Value'
-        WHEN is_extreme_outlier THEN 'Extreme Outlier (> 100 x10*9/L)'
-        WHEN confidence = 'NONE' AND conversion_reason = 'Excluded unit on this measurement type' THEN 'Excluded Unit (' || original_result_unit_code || ')'
-        WHEN confidence = 'NONE' AND inferred_value IS NULL THEN 'Value Out of Range After All Conversions'
-        WHEN confidence = 'NONE' THEN 'Could Not Determine Valid Value'
+        WHEN is_extreme_outlier THEN 'Extreme Outlier (> 20 x10*9/L)'
+        WHEN inferred_value IS NULL THEN 'Excluded Unit (' || original_result_unit_code || ')'
```

SQLFluff (4.2.2)

[error] Boolean/null literals must be lower case.

(CP04)